app/classifier/tasks/survey/choice.spec.js

```diff
@@ -29,17 +29,28 @@ describe('Choice', function () {
   let wrapper;
   let answer;
 
-  describe('with single answer questions', function () {
-    beforeEach(function () {
-      wrapper = shallow(<Choice
-        translation={task}
-        task={task}
-        annotation={annotation}
-        annotationValue={annotationValue}
-        choiceID='ar'
-      />);
-      answer = wrapper.find('input[name="ho"][value="two"]');
+  beforeEach(function () {
+    wrapper = shallow(<Choice
+      translation={task}
+      task={task}
+      annotation={annotation}
+      annotationValue={annotationValue}
+      choiceID='ar'
+    />);
+    answer = wrapper.find('input[name="ho"][value="two"]');
+  });
+
+  describe('with confused with options', function () {
+    it('should render the confusions with appropriate labels', function () {
+      const confusions = wrapper.find('.survey-task-confusions-modal');
```

Comment:
Maybe `wrapper.find(TriggeredModalForm)` here? We tend to avoid class name selectors in tests, although I can't remember why.